doc/api/test.md

```diff
@@ -521,7 +521,7 @@ test('mocks setTimeout to be executed synchronously without having to actually w
   const fn = mock.fn();
 
   // Optionally choose what to mock
-  mock.timers.enable(['setTimeout']);
+  mock.timers.enable({ timersToEnable: ['setTimeout'] });
```

Comment:
`mock.timers.enable` with `timersToEnable` seem redundant. What if we put `mock.timers.enable({ apis: ['setTimeout']})`

Reply:
@benjamingr suggested that we used the same as Jest uses with `toEnable`, what do you think?

Reply:
he said `toFake` for Date only, hasn't he?

Reply:
ah, you meant Sinon. Yeah, `toFake` or `toMock` seems good

Reply:
Erick's suggestion actually sounds better to me `enable`.

(toMock isn't great since we're not actually mocking anything, it's fake timers sunder the mock subsystem)

Reply:
I'll use the `apis` version then, also makes sense for me